Repositories/WorkoutRepository.cs

```diff
@@ -0,0 +1,101 @@
+using FitnessTrackerApi.Data;
+using FitnessTrackerApi.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace FitnessTrackerApi.Repositories;
+
+/// <summary>
+/// Репозиторий для работы с сущностью тренировки.
+/// Осуществляет CRUD операции для объектов <see cref="Workout"/>.
+/// </summary>
+public class WorkoutRepository : IWorkoutRepository
+{
+    private readonly ApplicationDbContext _context;
+
+    /// <summary>
+    /// Инициализирует новый экземпляр репозитория тренировок.
+    /// </summary>
+    /// <param name="context">Экземпляр контекста базы данных <see cref="ApplicationDbContext"/>.</param>
+    public WorkoutRepository(ApplicationDbContext context)
+    {
+        _context = context;
+    }
+
+    /// <summary>
+    /// Получает все тренировки.
+    /// </summary>
+    /// <param name="cancellationToken">Токен для отмены операции.</param>
+    /// <returns>Список всех тренировок.</returns>
```

Comment:
```suggestion
    /// <returns>Список всех тренировок.</returns>
```
А вот эти summary убери. Пусть только в интерфейсе будет описание